Vic2ToHoI4/Source/HOI4World/Map/Railway.h

```diff
@@ -0,0 +1,38 @@
+#ifndef HOI4WORLD_MAP_RAILWAY_H
+#define HOI4WORLD_MAP_RAILWAY_H
+
+
+
+#include <stdexcept>
+#include <vector>
+
+
+
+namespace HoI4
+{
+
+class Railway
+{
+  public:
+	Railway(int level, const std::vector<int>& provinces): level_(level), provinces_(provinces)
+	{
+		if (provinces.size() < 2)
+		{
+			throw std::runtime_error("Railways must have at least two provinces.");
```

Comment:
Do a soft warning instead of dying. I can envision many scenarios with dubious mapping files that on paper should involve multiple provinces but end up with junk.

Reply:
Lengths of one or fewer should already be stopped by the overall algorithm. If it actually gets to this, it really should stop the converter cold.